admin_guide/production_sizing.rst

```diff
@@ -0,0 +1,38 @@
+.. _production_sizing:
+
+Production Sizing
```

Comment:
Now that we have our fancy new front-page, could we add this to the migration button page?

Reply:
(Added)